IBM-GTS-Workflows/PA-zOSMF-Workflows-ZFS/IBM-MF-AUTO-ZFS-DEFINE-AMS.xml

```diff
@@ -146,8 +190,8 @@ If you have any questions please reach out to Eduardo Rocha:&lt;br&gt;
 
 </requestBody>
             <expectedStatusCode>200</expectedStatusCode>
-            <username substitution="false">A089712</username>
-            <password substitution="false">cElDNDJsYXQ=</password>
+            <username substitution="true">${ams-user}</username>
+            <password substitution="true">JHthbXMtcGFzc30=</password>
```

Comment:
Should this be using the new password variable?

Reply:
I think I uploaded the wrong/unfinished file, I will cancel the PR